vaxrank/mutant_protein_fragment.py

```diff
@@ -166,3 +168,23 @@ def top_k_subsequences(
         subsequences = list(self.generate_subsequences(subsequence_length))
         subsequences.sort(key=sort_key)
         return subsequences[:k]
+
+    def predicted_effect(self):
+        effects = [self.variant.effect_on_transcript(t) for t in
+            self.supporting_reference_transcripts]
+        predicted_effect = top_priority_effect(effects)
+        return predicted_effect
+
+    def global_start_pos(self):
+        # position of mutation start relative to the full amino acid sequence
+        global_mutation_start_pos = self.predicted_effect().aa_mutation_start_offset
```

Comment:
Makes sense after chatting offline. I think for now we can use the reference as a proxy for self, which will work correctly most of the time, but write up an explanation of where that goes wrong and keep an eye out for such cases in the future/fix it in isovar at some point.